src/fillTank.js

```diff
@@ -1,39 +1,39 @@
-'use strict';
-
-/**
- * @typedef {Object} Vehicle
- * @property {number} maxTankCapacity
- * @property {number} fuelRemains
- *
- * @typedef {Object} Customer
- * @property {number} money
- * @property {Vehicle} vehicle
- *
- * @param {Customer} customer
- * @param {number} fuelPrice
- * @param {number} amount
- */
-function fillTank(customer, fuelPrice, amount = Infinity) {
-  const { vehicle } = customer;
-  const freeSpace = vehicle.maxTankCapacity - vehicle.fuelRemains;
-  const canBuy = customer.money / fuelPrice;
-  const requiredAmount = Math.min(amount, freeSpace, canBuy);
-  const roundedAmount = roundFuel(requiredAmount);
-
-  if (roundedAmount < 2) {
-    return;
-  }
-
-  customer.vehicle.fuelRemains += roundedAmount;
-  customer.money -= roundPrice(roundedAmount * fuelPrice);
-}
-
-function roundFuel(fuel) {
-  return Math.floor(fuel * 10) / 10;
-}
-
-function roundPrice(price) {
-  return Math.round(price * 100) / 100;
-}
-
-module.exports = { fillTank };
+'use strict';
+
+/**
+ * @typedef {Object} Vehicle
+ * @property {number} maxTankCapacity
+ * @property {number} fuelRemains
+ *
+ * @typedef {Object} Customer
+ * @property {number} money
+ * @property {Vehicle} vehicle
+ *
+ * @param {Customer} customer
+ * @param {number} fuelPrice
+ * @param {number} amount
+ */
+function fillTank(customer, fuelPrice, amount = Infinity) {
+  const { vehicle } = customer;
+  const freeSpace = vehicle.maxTankCapacity - vehicle.fuelRemains;
+  const canBuy = customer.money / fuelPrice;
+  const requiredAmount = Math.min(amount, freeSpace, canBuy);
+  const roundedAmount = roundFuel(requiredAmount);
+
+  if (roundedAmount < 2) {
+    return;
+  }
+
+  customer.vehicle.fuelRemains += roundedAmount;
+  customer.money -= roundPrice(roundedAmount * fuelPrice);
+}

```

Comment:
The function `fillTank` is doing too many things. It's calculating the amount of fuel that can be bought, rounding the fuel amount, adding fuel to the vehicle, and subtracting money from the customer. It would be better to break this function down into smaller functions, each with a single responsibility. This would make the code easier to understand and maintain.